frontend/src/components/dashboard/DashboardTransactions.jsx

```diff
@@ -1,94 +1,140 @@
 import Link from "next/link";
 
-import React from "react";
+import React, { Component } from "react";
 
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
 import { Row, Col } from "react-bootstrap";
 
 import Content from "../utils/Content";
-import TransactionsList from "../transactions/TransactionsList";
-
-export default ({ transactions }) => (
-  <Content
-    title={<h2>Recent Transactions</h2>}
-    icon={<TransactionIcon style={{ width: "22px" }} />}
-    size="medium"
-    border={false}
-    className="dashboard-transactions"
-  >
-    <Row className="px-0">
-      <Col xs="1" className="pr-0">
-        <div className="dashboard-blocks-hr-parent">
-          <div className="dashboard-blocks-hr" />
-        </div>
-      </Col>
-      <Col xs="11" className="px-0 dashboard-transactions-list">
-        <TransactionsList
-          transactions={transactions}
-          viewMode="compact"
-          reversed
-        />
-        <Row noGutters>
-          <Col xs="1" className="dashboard-transactions-icon-col" />
-          <Col xs="6">
-            <Link href="transactions">
-              <a className="dashboard-footer">View All</a>
-            </Link>
+import ActionsList from "../transactions/ActionsList";
+
+import TransactionsApi from "../../libraries/explorer-wamp/transactions";
+import FlipMove from "react-flip-move";
+
+export default class extends Component {
+  state = {
+    transactions: this.props.transactions
+  };
+
+  componentDidMount() {
+    this.regularFetchInfo();
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+    this.timer = false;
```

Comment:
Let's use `null` instead of boolean value to indicate that something is not set yet.

Reply:
got it